src/app/_mocked/mock-line.component.ts

```diff
@@ -87,6 +87,10 @@ export class MockLineComponent {
     console.log('MockLineComponent addSeries');
   }
 
+  addSeriesData(_, __, ___): void {
```

Comment:
this way of naming parameters is quite crazy, is just every time I see them I need to get use to them.